### PR TITLE
fix(home-manager): add file for yazi syntax highlighting

### LINKED ISSUE #111: yazi: theme installs but no synatx highlighting
The yazi theme decalres the synax highligting to `~/.config/yazi/theme.tmTheme`
see https://github.com/catppuccin/yazi/blob/0846aed69b2a62d29c98e100af0cf55ca729723d/themes/mocha.toml#L32

But the flake does not provide that file, this could be solved by useing the bat themes' .tmTheme file.

### PR DESCRIPTION
**Pull request by michaelBelsanti**
Fixes #111

### CODE REVIEW

**Review by isabelroses — APPROVED** [review]

**Review by getchoo — APPROVED** [review]
thanks!

File: modules/home-manager/yazi.nix [hunk 1]
```diff
@@ -14,5 +14,6 @@ in
 
   config = lib.mkIf enable {
```
Comment (getchoo) [inline modules/home-manager/yazi.nix:15]:
```suggestion
  config = lib.mkIf enable {
    assertions = [
      (lib.ctp.assertXdgEnabled "yazi")
    ];
```
oops, might have been a bit too eager 😆 
an assertion will need to be used here to make sure `xdg` is enabled

**Review by getchoo — CHANGES_REQUESTED** [review]
oops, was a little bit too quick there.

File: modules/home-manager/yazi.nix (hunk 1 above)
Comment (michaelBelsanti) [reply modules/home-manager/yazi.nix:15]: good catch, forgot that you could disable xdg

**Review by getchoo — APPROVED** [review]
thanks!

**Pull request merged**